ui/lib/core/addon/components/alert-banner.js

```diff
@@ -14,13 +14,15 @@ import layout from '../templates/components/alert-banner';
  *
  * @param type=null {String} - The banner type. This comes from the message-types helper.
  * @param [message=null {String}] - The message to display within the banner.
+ * @param [title=null {String}] - A title to show above the message. If this is not provided, there are default values for each type of alert.
```

Comment:
oooh thank you. can you re-generate the `alert-banner.md` so the component documentation in storybook is updated too? 💅

Reply:
Oh yep!